acados/ocp_nlp/ocp_nlp_cost_ls.c

```diff
@@ -607,22 +661,51 @@ void ocp_nlp_cost_ls_update_qp_matrices(void *config_, void *dims_,
 
     int nx = dims->nx;
     int nu = dims->nu;
+    int nz = dims->nz;
     int ny = dims->ny;
     int ns = dims->ns;
 
     // initialize hessian of lagrangian with hessian of cost
     blasfeo_dgecp(nu + nx, nu + nx, &memory->hess, 0, 0, memory->RSQrq, 0, 0);
 
+    // copy Cyt into Cyt_tilde
+    blasfeo_dgecp(nu + nx, ny, &model->Cyt, 0, 0, &work->Cyt_tilde, 0, 0);
```

Comment:
Avoid unnecessary copy for the most common case of `nz=0`, and keep the old code with an `if ... else ...`

Reply:
addressed in 9319a5.